server/ansible/pbench-server-config.yml

```diff
@@ -0,0 +1 @@
+/home/dbutenho/pbench-server-config.yml
```

Comment:
This looks like an odd thing to commit to the repo...is this file supposed to be include with this PR?

Reply:
I missed this one; yeah, I sometimes use this to configure post-installation, and I forgot to delete it. I didn't even notice when I looked through the inventory.